src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/ccd/CcdClient.java

```diff
@@ -69,58 +77,89 @@ public void completeAwaitingDcnProcessing(
         );
     }
 
-    private StartEventResponse startEvent(
+    private StartEventResponse startCompleteAwaitingDcnProcessing(
         CcdAuthenticator authenticator,
         String jurisdiction,
         String caseTypeId,
-        String caseRef,
-        String eventTypeId
+        String caseRef
     ) {
-        StartEventResponse response = ccdApi.startEventForCaseWorker(
-            authenticator.getUserToken(),
-            authenticator.getServiceToken(),
-            authenticator.getUserDetails().getId(),
-            jurisdiction,
-            caseTypeId,
-            caseRef,
-            eventTypeId
-        );
-
-        log.info(
-            "Started event {} for case {}. Jurisdiction: {}. Case type ID: {}",
-            eventTypeId,
-            caseRef,
-            jurisdiction,
-            caseTypeId
-        );
-
-        return response;
+        try {
+            StartEventResponse response = ccdApi.startEventForCaseWorker(
+                authenticator.getUserToken(),
+                authenticator.getServiceToken(),
+                authenticator.getUserDetails().getId(),
+                jurisdiction,
+                caseTypeId,
+                caseRef,
+                COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID
+            );
+
+            log.info(
+                "Started event {} for case {}. Jurisdiction: {}. Case type ID: {}",
+                COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID,
+                caseRef,
+                jurisdiction,
+                caseTypeId
+            );
+
+            return response;
+        } catch (FeignException ex) {
+            debugCcdException(ex, "Failed to call 'startCompleteAwaitingDcnProcessing'");
+            throw new CcdCallException(
+                format(
+                    "Internal Error: Failed starting event in CCD for completing payment DCN processing "
+                        + "case: %s Error: %s", caseRef,
```

Comment:
Perhaps I should've mentioned it: not sure `Internal Error:` serves any purpose here. Same with another one below